yb-voyager/src/srcdb/source.go

```diff
@@ -86,12 +87,34 @@ func (s *Source) IsOracleCDBSetup() bool {
 
 func (s *Source) ApplyExportSchemaObjectListFilter() {
 	allowedObjects := utils.GetSchemaObjectList(s.DBType)
-	if s.StrExportObjectTypesList == "" {
+
+	if s.StrExportObjectTypesList == "" && s.StrExcludeObjectTypesList == "" {
 		s.ExportObjectTypesList = allowedObjects
 		return
 	}
-	expectedObjectsSlice := strings.Split(s.StrExportObjectTypesList, ",")
+
+	if s.StrExcludeObjectTypesList != "" {
+		excludedObjectsSlice := utils.CsvStringToSlice(s.StrExcludeObjectTypesList)
+		s.ExportObjectTypesList = lo.Filter(allowedObjects, func(objType string, _ int) bool { return !utils.ContainsString(excludedObjectsSlice, objType) })
+		filteredObjects := lo.Filter(excludedObjectsSlice, func(objType string, _ int) bool { return !utils.ContainsString(allowedObjects, objType) })
+		printAndCheckFilteredObjects(filteredObjects, s)
+		return
+	}
+
+	expectedObjectsSlice := utils.CsvStringToSlice(s.StrExportObjectTypesList)
 	s.ExportObjectTypesList = lo.Filter(allowedObjects, func(objType string, _ int) bool { return utils.ContainsString(expectedObjectsSlice, objType) })
+	filteredObjects := lo.Filter(expectedObjectsSlice, func(objType string, _ int) bool { return !utils.ContainsString(allowedObjects, objType) })
+	printAndCheckFilteredObjects(filteredObjects, s)
+}
+
+func printAndCheckFilteredObjects(filteredObjects []string, s *Source) {
+	if len(filteredObjects) > 0 {
+		utils.PrintAndLog("Ignoring invalid object types: %s\n", strings.Join(filteredObjects, ", "))
+	}
+	if !utils.ContainsString(s.ExportObjectTypesList, "TABLE") && utils.ContainsString(s.ExportObjectTypesList, "INDEX") {
+		s.ExportObjectTypesList = lo.Filter(s.ExportObjectTypesList, func(objType string, _ int) bool { return objType != "INDEX" })
```

Comment:
Talk to Sanyam to see if there are any other types that you need to skip.

Reply:
Yeah he only asked me to add this